src/plugins/data_table/column_types/attr.js

```diff
@@ -4,7 +4,7 @@ app.view.dataTable.columnType(function(columnType) {
   });
 
   columnType.nodeMatcher(function(config) {
-    return "." + Backdraft.Utils.toCSSClass(config.title);
+    return "." + Backdraft.Utils.toCSSClass(config.attr || config.title);
```

Comment:
ah, Mike's comment is a good one here though, I led you slightly astray yesterday Alberto.  Yes, definitely true that some tables don't use config.attr at all, and that is a main reason for the redo of the implementation here.. but this `column_types/attr.js` should only be used on columns where there **is** an attr. (See lines 2~4).

So in this file, I think it is correct / more accurate to just use config.attr on this line and line 16 below.